tests/timezones/tests.py

```diff
@@ -265,6 +236,20 @@ def test_raw_sql(self):
             [event],
             transform=lambda d: d)
 
+    def test_cursor_execute_accepts_naive_datetime(self):
```

Comment:
They are in different `TestCase`s. The first one is in `LegacyDatabaseTests` and the second on in `NewDatabaseTests`. `USE_TZ` is `False` and `True` respectively. The two classes have a very similar set of tests but make different assertions.